types/relay-test-utils/test/index.tsx

```diff
@@ -111,7 +111,9 @@ function environmentTests() {
         <Modal queryRef={queryRef} />
     </RelayEnvironmentProvider>;
 
-    const operation = environment.mock.findOperation(operation => operation.root.node === userQuery);
+    const operation = environment.mock.findOperation(operation =>
```

Comment:
this appears to have been a mistake in master that these changes, for some reason, uncovered. Anyway, this appears to be correct now: `userQuery` is cast to `ConcreteRequest` (which is its concrete type) and `.operation` has type `NormalizationOperation`. `operation.root.node` has type `NormalizationSelectableNode`, which includes `NormalizationOperation`